tests/integration/conftest.py

```diff
@@ -69,3 +69,24 @@ def local_pypiserver() -> Iterator[None]:
             raise RuntimeError("pypiserver didn't start fast enough")
 
         yield
+
+
+def does_not_test_any_affected_pms(test: pytest.Item, affected_pms_to_test: set[str]) -> bool:
+    pm_under_test = utils.tested_object_name(Path(test.fspath))
```

Comment:
`fspath` has a type of `LEGACY_PATH`, which will be deprecated at some point. You can use just `path` without type conversion
https://github.com/pytest-dev/pytest/blob/72c682ff9773ad2690711105a100423ebf7c7c15/src/_pytest/nodes.py#L139

Reply:
Good point, thank you, will update in the next revision.